crates/cxx-qt-lib/src/gui/qcolor.cpp

```diff
@@ -16,11 +16,11 @@
 // compiler padding this results in a sizeof 16.
 // https://code.qt.io/cgit/qt/qtbase.git/tree/src/gui/painting/qcolor.h?h=v5.15.6-lts-lgpl#n262
 // https://code.qt.io/cgit/qt/qtbase.git/tree/src/gui/painting/qcolor.h?h=v6.2.4#n237
-assert_alignment_and_size(QColor,
-                          alignof(::std::size_t),
-                          sizeof(::std::int32_t) +
-                            (sizeof(::std::uint16_t) * 5) +
-                            2 /* compiler padding */);
```

Comment:
note that on the Rust side we have
```rust
#[repr(C)]                                                                                                                                                                                                                             
pub struct QColor {                                                                                                                                                                                                                    
    _cspec: MaybeUninit<i32>,                                                                                                                                                                                                          
    _ct: MaybeUninit<[u16; 5]>,                                                                                                                                                                                                        
    _padding: MaybeUninit<u16>,                                                                                                                                                                                                        
} 
```

Wonder if we should not have the padding here on the Rust side and leave the compiler to fill any padding automatically?